src/frontend/src/icp/components/send/IcSendForm.svelte

```diff
@@ -19,12 +19,15 @@
 	export let amount: OptionAmount = undefined;
 	export let networkId: NetworkId | undefined = undefined;
 	export let source: string;
+	export let simplifiedForm = false;
```

Comment:
Furthermore this would comply with the existing logic in `EthSendTokenWizard` and would also have the advantages to inherit the fact that the destination would not be displayed in the review step as well.

BUT, I might be overseeing something. Let me know what you think?

Reply:
`destinationEditable` is also not the best name, because it's not just the destination field that is not displayed but also the source field.
That's why i chose the name `simplifiedForm`.

Reply:
Thanks for the explanation. My question probably originated as I missed a piece of information in my previous review. Let's continue with it then, agree it serves more than one purpose.